src/pages/admin-ui-sdk/publish.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Prepare your app for production
+description: Learn how to prepare your app for non-local testing and for production.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# Prepare your app for production
+
+After you have tested your app locally, you are ready to publish your app to your Production environment. When you have accomplished this, your app is available in the Adobe App Registry. Adobe Commerce will then be able fetch the information it needs to augment the Admin.
```

Comment:
Can we reword this to "you are ready to submit your app for approval. Once the app has been approved, it will e available in the Adobe App Registry. The Admin UI SDK will then be able to fetch the information it needs to create the app UI in Adobe Commerce"